translations/localify/ui.json

```diff
@@ -1997,5 +1998,158 @@
     "このレースで{0}着以内になる": "Place top {0} in this race",
     "現在選択しているウマ娘を編成すると、\n上記のウマ娘がチームから外れますがよろしいですか？": "Adding these horsegirls will remove the following \nones from the team, are you sure?",
     "出走履歴がありません": "Race history lacking",
-    "編成時に対象キャラをハイライトする": "Highlight these chars during set up"
+    "編成時に対象キャラをハイライトする": "Highlight these chars during set up",
+    "<chrname>は、\nなにか得るものがあったようだ！": "<chrname> seems to have gained something! ",
+    "その時、ふと閃いた！\nこのアイディアは、<chrname>との\nトレーニングに活かせるかもしれない！": "Just then, I had an idea! \nThis might help <chrname>'s training!",
+    "<chrname>の成長につながった！": "It helped <chrname> grow! ",
+    "お休み": "Rest",
+    "1月": "Jan",
+    "2月": "Feb",
+    "3月": "Mar",
+    "4月": "Apr",
+    "5月": "May",
+    "6月": "Jun",
+    "7月": "Jul",
+    "8月": "Aug",
+    "9月": "Sep",
+    "10月": "Oct",
+    "11月": "Nov",
+    "12月": "Dec",
```

Comment:
> saw that Current ads expire in is translated in static.json... and the full Current ads expire in <time> texts that static_dump pops out had it translated. 

Interesting. I never quite cared enough to check this out fully but the instructions did mention removing the dicts to be sure. If this is what happens then the change of removing that part in the guide (in this pr) should be reverted too.

> If you'd like, I could remove this for now for the sake of not letting the PR get too stale, and maybe you or someone who knows umacruise could look into it later.

Hmm, sure. Maybe make a separate PR for it and it can be checked in time.